doc/en/components/charts/chart-features-markers.md

```diff
@@ -7,7 +7,57 @@ mentionedTypes: []
 
 # $PlatformShort$ Chart Markers
 
-<!-- talk about marker types, marker templates, marker appearance
+There are many areas of the $ProductName$ category chart component that can be customized, such as the markers, chart titles and subtitles, and the chart types.
 
-data-chart-series-markers.md
--->
\ No newline at end of file
+<code-view style="height: 500px" 
+           data-demos-base-url="{environment:dvDemosBaseUrl}" 
+           iframe-src="{environment:dvDemosBaseUrl}/charts/category-chart-marker-options" 
+           alt="$PlatformShort$ Configuration Options Example" 
+           github-src="charts/category-chart/marker-options">
+</code-view>
+
+<div class="divider--half"></div>
+
+## Configuring Chart Markers
+
+Markers are visual elements that display the values of data points in the category chart component's plot area. Markers help your end-users immediately identify a data point's value even if the value falls between major or minor grid lines.
+
+The appearance of chart markers is managed through the `MarkerType`, `MarkerBrushes`, `MarkerOutlines` properties of the category chart, as demonstrated in the following code example.
+
+```razor
+ <CategoryChart Height="500px" Width="700px"
+    DataSource="@DataSource"
+    MarkerType="MarkerType.Diamond"
+    MarkerBrushes="red"
+    MarkerOutlines="yellow"/>
+```
+
+```html
+<igx-category-chart
+    [dataSource]="data"
+    width="700px"
+    height="500px"
+    markerType="diamondMarker"
+    markerBrushes="red"
+    markerOutlines="yellow">
+</igx-category-chart>
```

Comment:
"diamondMarker" is invalid value for markerType. Use "Diamond"

Reply:
pending